test/suite/unit/Exception/StaleUpdateException.spec.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Exception;
+
+use Rinq\Ident\PeerId;
+use Rinq\Ident\Reference;
+use Rinq\Ident\SessionId;
+
+describe('StaleUpdateException', function () {
+    beforeEach(function () {
+        $this->reference = Reference::create(SessionId::create(PeerId::create(111, 222), 333), 444);
+        $this->subject = new StaleUpdateException($this->reference);
+    });
+
+    describe('->construct()', function () {
+        it('should construct a meaningful exception message', function () {
+            expect($this->subject->getMessage())->to->equal(
+                'Can not update or close 6F-00DE.333@444, the session has ' .
+                'been modified since that revision.'
```

Comment:
This is *really* picky, but if you're gonna split strings, it should probably be tabbed in on the second line to make it clear that it's part of the same expression.  My personal preference is to simply not split them, but it's your call.